### PR TITLE
VSAC, manifest loader reorg, ruff

### PR DESCRIPTION
**Pull request by dogversioning**
This PR makes the following changes:
- Removed the interim directory based on manifest-based study loading
  - this created some churn in this PR - i'll try to call it out.
- Added VSAC python module to vocab/analyze
- ruff check/format run

### CODE REVIEW

File: cumulus_library_opioid/counts.py
Comment (dogversioning) [inline cumulus_library_opioid/counts.py:1]: This is just a move, dont know why it didn't get picked up

File: cumulus_library_opioid/valueset.py
Comment (dogversioning) [inline cumulus_library_opioid/valueset.py:1]: Ruff only, no functional changes

File: cumulus_library_opioid/vocab/constants.py
Comment (dogversioning) [inline cumulus_library_opioid/vocab/constants.py:1]: ruff only, no functional changes

File: cumulus_library_opioid/vocab/generate/file_bash.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import unittest
+from cumulus_library_opioid.vocab.constants import CURATED_LIST
```
Comment (dogversioning) [inline cumulus_library_opioid/vocab/generate/file_bash.py:2]: import path is only functional change in this file

File: cumulus_library_opioid/vocab/generate/file_sql.py
Comment (dogversioning) [inline cumulus_library_opioid/vocab/generate/file_sql.py:1]: ruff only, no changes

File: cumulus_library_opioid/vocab/jaccard/jaccard.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import unittest
-from cumulus_library_opioid.opioid.vocab.constants import CURATED_LIST
+from cumulus_library_opioid.vocab.constants import CURATED_LIST
```
Comment (dogversioning) [inline cumulus_library_opioid/vocab/jaccard/jaccard.py:2]: import path is only functional change in this file

File: cumulus_library_opioid/vocab/medrt/medrt.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
+import unittest
+from cumulus_library_opioid.vocab.generate.file_sql import (
```
Comment (dogversioning) [inline cumulus_library_opioid/vocab/medrt/medrt.py:2]: import path is only functional change in this file

File: cumulus_library_opioid/vocab/analyze/analyze.sh [hunk 4]
```diff
@@ -13,10 +13,9 @@ fi
 echo "Using $CURATED.tsv"
 $mysql_table_schema -e "call version('${CURATED}', 'results.sh:begin')"
 
-# TODO: links with (ln -s) should be replaced with Matt's true VSAC downloader.
-# Each "curated" source is a TSV file with two cols, RXCUI and STR
+python vsac.py $CURATED
 rm -f curated.tsv
-ln -s infile/$CURATED.tsv curated.tsv
+ln -s ../data/$CURATED.tsv curated.tsv
```
Comment (dogversioning) [inline cumulus_library_opioid/vocab/analyze/analyze.sh:18]: @comorbidity this should be the only functional. change to the current bash workflow

**Review by mikix — APPROVED** [review]
Sure - tried to skim through just the vsac stuff and it seemed not crazy :smile: :+1:

**Pull request merged**